test/e2e/appstudio_test.go

```diff
@@ -0,0 +1,203 @@
+package e2e
```

Comment:
Maybe we should have an `appstudio` package within the `e2e` package? Then we can put all our appstudio-specific tests there?

Reply:
Technically, all e2e tests are used also for AppStudio case - not only Dev Sandbox. 
I would just avoid using the term AppStudio as it could confuse some ppl. I would vote for naming it `proxy_test.go` for now. We can change it later to `AppStudio` if there is any need in using it.

Reply:
Renamed to `proxy_test.go` 75080f9ffbcd1671fbb15590b41205ef7a02c5ec